src/styles/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__ul {
+    list-style: none;
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__li {
```

Comment:
Using tag names as class names is considered bad practice because it creates tight coupling between the HTML structure and the CSS styling, which makes the code less maintainable and flexible.
For example, if you use the class name "div" to style a specific div element, and you later decide to change that element to a different tag, such as an article or section, you will have to update the class name throughout the entire codebase. This can be time-consuming and error-prone, especially in larger projects.
In addition, using tag names as class names can also cause conflicts and confusion when working with multiple stylesheets or frameworks, as different developers may use the same tag names as class names in different contexts.
To avoid these issues, it's generally recommended to use more descriptive class names that reflect the purpose or function of the element being styled, rather than its tag name. This promotes a separation of concerns between the HTML and CSS, making the code more modular, maintainable, and adaptable to future changes.